app/components/views/TicketsPage/MyTicketsTab/TicketInfoCard.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import TicketCard from "./TicketCard";
+import ExpandedInfo from "./ExpandedInfo";
+import Balance from "Balance";
+import { Tooltip } from "shared";
+import { tsToDate } from "helpers/dateFormat";
+import { FormattedMessage as T } from "react-intl";
+
+const statusTxt = {
+  "unknown": <T id="ticket.status.unknown" m="unknown" />,
+  "unmined": <T id="ticket.status.unmined" m="unmined" />,
+  "immature": <T id="ticket.status.immature" m="immature" />,
+  "live": <T id="ticket.status.live" m="live" />,
+  "voted": <T id="ticket.status.voted" m="voted" />,
+  "missed": <T id="ticket.status.missed" m="missed" />,
+  "expired": <T id="ticket.status.expired" m="expired" />,
+  "revoked": <T id="ticket.status.revoked" m="revoked" />,
+};
+
+const TicketInfoCard = ({ ticket, onClick, expanded }) => {
+
+  const className = "ticket-info-card" + (expanded ? " is-expanded" : "");
+  const rewardTipText = <T id="ticket.rewardCalc"
+    m={`Investment: {investment}
+    Transaction Fee: {txFee}
+    Pool Fee: {poolFee}
+    Ticket Reward: {reward}
+    ROI: {roi, number, precise-percent}`}
+    values={{
+      investment: <Balance amount={ticket.ticketInvestment || 0} />,
+      txFee: <Balance amount={ticket.ticketTxFee || 0} />,
+      poolFee: <Balance amount={ticket.ticketPoolFee || 0} />,
+      reward: <Balance amount={ticket.ticketReward || 0} />,
+      roi: ticket.ticketROI
+    }} />;
+
+  let timeToLeaveTipText;
+  if (ticket.leaveTimestamp) {
+    const days = Math.ceil((ticket.leaveTimestamp - ticket.enterTimestamp) / 86400);
+    timeToLeaveTipText = <T id="ticket.daysToLeave"
+      m="~ {days, plural, one {# day} other {# days}} from buying until {status}"
```

Comment:
Not really happy with this copy, if anyone has a better suggestion.

Reply:
I think it's clear what you're trying to say.  Can you add a note for the 86400 though?  Like `// 86400 seconds = 1 day`  and maybe even do `24 * 60 * 60` instead of 86400